kubecost/templates/_helpers.tpl

```diff
@@ -548,21 +548,24 @@ federated storage config helpers
   {{- end }}
 {{- end -}}
 
+    {{/*
+    NOTE: added kubecostModel for backward compatibility
+    */}}
 {{- define "kubecost.federatedStorage.config" }}
   {{- if (.Values.kubecostModel).federatedStorageConfig -}}
-    {{- (.Values.kubecostModel).federatedStorageConfig -}}
-  {{- else if (.Values.federatedStorage).config }}
-    {{- (.Values.federatedStorage).config -}}
+    {{- toYaml (.Values.kubecostModel).federatedStorageConfig -}}
+  {{- else if (.Values.federatedStorage).config -}}
+    {{- toYaml (.Values.federatedStorage).config -}}
```

Comment:
Ditto here. It looks like our current values file is expecting `.Values.federatedStorage.config` to be a string?

https://github.com/kubecost/kubecost/blob/87ca5777d1e494e8244c061b7c63e5e7c64d1b76/kubecost/values.yaml#L535